all_fruits.py

```diff
@@ -0,0 +1,163 @@
+import random, pygame, sys, math
+from pygame.locals import *
+
+white = (255, 255, 255)
+green = (0, 255, 0)
+
+class Fruit(pygame.sprite.Sprite):
+    def __init__(self, screen, image_name):
+        # Call the parent class (Sprite) constructor
+        pygame.sprite.Sprite.__init__(self)
```

Comment:
You can use `super().__init__(self)` instead of `pygame.sprite.Sprite.__init__(self)`. This makes the code less fragile (you can can change `class Fruit(pygame.sprite.Sprite)` to `class Fruit(MySprite):`, where `MySprite` is some intermediate subclass of `pygame.sprite.Sprite`, without needing to search `Fruit`'s methods for occurrences of `pygame.sprite.Sprite`).

This also allows you to eliminate the comment, since the code is now self-documenting.